invenio_administration/assets/semantic-ui/js/invenio_administration/src/components/String.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+class String extends React.Component {
```

Comment:
I think we can omit this component, since we have the fallback in the formatter above